xrpl/models/transactions/xchain_add_attestation.py

```diff
@@ -0,0 +1,212 @@
+"""Model for a XChainAddAttestation transaction type."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Type, Union
+
+from typing_extensions import Literal
+
+from xrpl.models.amounts import Amount
+from xrpl.models.base_model import BaseModel
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import require_kwargs_on_init
+from xrpl.models.xchain_bridge import XChainBridge
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class XChainClaimAttestationBatchElement(BaseModel):
```

Comment:
These helper models will be simplified after #421 is merged

Reply:
Is this an outstanding TODO now that it's merged?